### ISSUE TITLE
Email setup

### ISSUE THREAD

**Issue by CarlQLange** [arunoda/meteor-up-legacy#430]
Hey there,

Is there a nice way to set up an email server alongside mongo etc so that I can send emails? Is this already built-in to mup? If not, could anyone advise me on what I need to do so my server will send me things?

Cheers,
Carl

**Comment by stephentcannon** [comment]
See http://docs.meteor.com/#/full/email

This will allow you to send mail via SMTP.

Then use any SMTP provider like MailGun etc...

**Comment by CarlQLange** [comment]
Ah, I must have been unclear - I meant setting up an SMTP server on a server when setting up with `mup init`. Mongo and Node etc are installed by the tool - why not postfix or similar as well?

**Comment by stephentcannon** [comment]
I believe it is out of scope for mup.  Otherwise you could then ask why not nginx or haproxy or something else.  I do not think mup is meant as a devops management system but rather a simple way to deploy your application.  A meteor app is reliant on mongo and node but not postfix.  If you really want it I imagine you could submit a pr.

**Comment by CarlQLange** [comment]
I would say it is at least partially reliant on email - quite a large proportion of webapps use email in some form or another, and to get a completely deployed, production app using mup, email is definitely a part of that. I do see your point in regards to the slippery slope problem - however this is a feature that meteor natively supports that mup does not handle.

Given enough time (eg a few months, depending on the amount of Actual Paying Work I get) I would happily write a PR for this. I was mainly curious if it had been discussed before :)

**Comment by CarlQLange** [comment]
I've actually just browsed through the source and noticed they're just shell scripts - if I have the time in the next few days I'll write a PR, didn't realise how little was involved to add this.

**Comment by CarlQLange** [comment]
Also re: nginx: https://github.com/arunoda/meteor-up/pull/275 :)

**Comment by stephentcannon** [comment]
Agreed.  I have had similar thoughts RE: nginx in mup but felt better to manage it with something like Puppet or Chef though.

**Comment by stephentcannon** [comment]
Check out 

https://github.com/arunoda/meteor-up/tree/mupx#

Probably a better solution if you go with docker.  

Specifically see the nginx implementation now.
